test/units/cli/test_doc.py

```diff
@@ -27,6 +27,11 @@
     'IBM(International Business Machines)': 'IBM(International Business Machines)',
     'L(the user guide, https://docs.ansible.com/)': 'the user guide <https://docs.ansible.com/>',
     'R(the user guide, user-guide)': 'the user guide',
+    # de-rsty refs and anchors
+    'yolo :ref:`my boy` does stuff': 'yolo website for `my boy` does stuff',
+    '.. seealso:: Something amazing': 'See website for: Something amazing',
+    '.. seealso:: Troublesome multiline\n Stuff goes htere': 'See website for: Troublesome multiline\n Stuff goes htere',
+    '.. notes:: boring stuff': 'Notes: boring stuff',
```

Comment:
Same as above: I'm not aware of any Sphinx extension that we use defining the plular `.. notes::`.